superbench/benchmarks/micro_benchmarks/disk_performance.py

```diff
@@ -126,16 +126,58 @@ def add_parser_arguments(self):
                     help='Number of threads in %s test.' % io_str,
                 )
 
-    def _preprocess(self):
+        self._parser.add_argument(
+            '--numa_node',
+            type=int,
+            required=False,
+            help='The index of NUMA node.',
+        )
+        self._parser.add_argument(
+            '--verify',
+            type=str,
+            required=False,
+            help='Verification method specified for fio --verify flag.',
+        )
+
+    def __get_arguments_from_env(self):
+        """Read environment variables from runner used for parallel and fill in block_devices and numa_node_index.
+
+        Get 'PROC_RANK'(rank of current process) 'BLOCK_DEVICES' 'NUMA_NODES' environment variables
+        Get block_devices and numa_node_index according to 'BLOCK_DEVICES'['PROC_RANK'] and 'NUMA_NODES'['PROC_RANK']
+        - BLOCK_DEVICES should be comma-separated strings, each being colon-separated block devices.
+          E.g., "/dev/nvme0n1:/dev/nvme1n1,/dev/nvme2n1:/dev/nvme3n1".
+        - NUMA_NODES should be comma-separated numbers. E.g., "0,1".
+        Note: The config from env variables will overwrite the configs defined in the command line.
+        """
+        try:
+            if os.getenv('PROC_RANK'):
+                rank = int(os.getenv('PROC_RANK'))
+                if os.getenv('BLOCK_DEVICES'):
+                    self._args.block_devices = os.getenv('BLOCK_DEVICES').split(',')[rank].split(':')
```

Comment:
why use env rather than argument, the parser is not robust

Reply:
Change to ise block device indices in envs, block device strings will be passed from argument